app/templates/_global_components.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- Service-backed components --}}
+
+{{offer-sort-overlay
```

Comment:
I might be wrong, but I think this is only used in one place of the app (right ?)
So you could just add it to `search.hbs` without having it as an app level component.